ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -227,7 +246,96 @@ const swordQuiverOutputStrings = {
   },
 } as const;
 
+const conductorCurrentStringsNoStrat = {
+  remoteCurrent: {
+    en: 'Far Cone on You',
+    de: 'Fern-Kegel auf DIR',
+    fr: 'Cône éloigné sur Vous',
+    ja: '自分から遠い人に扇範囲',
+    cn: '远雷点名',
+    ko: '원거리 화살표 대상자',
+  },
+  proximateCurrent: {
+    en: 'Near Cone on You',
+    de: 'Nah-Kegel auf DIR',
+    fr: 'Cône proche sur Vous',
+    ja: '自分から近い人に扇範囲',
+    cn: '近雷点名',
+    ko: '근거리 화살표 대상자',
+  },
+  spinningConductorSupport: {
+    en: 'Small AoE on You',
+    de: 'Kleine AoE auf DIR',
+    fr: 'Petite AoE sur Vous',
+    ja: '自分に小さい円範囲',
+    cn: '小钢铁点名',
+    ko: '작은 원형징 대상자',
+  },
+  spinningConductorDPS: {
+    en: 'Small AoE on You',
+    de: 'Kleine AoE auf DIR',
+    fr: 'Petite AoE sur Vous',
+    ja: '自分に小さい円範囲',
+    cn: '小钢铁点名',
+    ko: '작은 원형징 대상자',
+  },
+  roundhouseConductorSupport: {
+    en: 'Donut AoE on You',
+    de: 'Donut AoE auf DIR',
+    fr: 'Donut sur Vous',
+    ja: '自分にドーナツ範囲',
+    cn: '月环点名',
+    ko: '도넛징 대상자',
+  },
+  roundhouseConductorDPS: {
+    en: 'Donut AoE on You',
+    de: 'Donut AoE auf DIR',
+    fr: 'Donut sur Vous',
+    ja: '自分にドーナツ範囲',
+    cn: '月环点名',
+    ko: '도넛징 대상자',
+  },
+  colliderConductor: {
+    en: 'Get Hit by Cone',
+    de: 'Werde vom Kegel getroffen',
+    fr: 'Encaissez un cône',
+    ja: '扇範囲に当たって',
+    cn: '吃雷',
+    ko: '화살표 장판 맞기',
+  },
+} as const;
+
+const conductorCurrentStringsDNStrat = {
+  remoteCurrent: {
+    en: 'Front of Middle (Far Cone)',
+  },
+  proximateCurrent: {
+    en: 'Front of Middle (Near Cone)',
+  },
+  spinningConductorSupport: {
+    en: 'Left Rivet (Small AoE)',
+  },
+  spinningConductorDPS: {
+    en: 'Right Rivet (Small AoE)',
+  },
+  roundhouseConductorSupport: {
+    en: 'Left Rivet (Donut AoE)',
+  },
+  roundhouseConductorDPS: {
+    en: 'Right Rivet (Donut AoE)',
+  },
+  colliderConductor: {
+    en: 'Middle Rivet (Get Hit by Cone)',
```

Comment:
Sorta wondering if 'rivet' is really necessary? It seems like left or right would communicate the necessary information (or perhaps even 'Front Left' or 'Front Right'.  I think 'Middle Rivet' may also be confusing since it doesn't clearly communicate position relative to the `current` debuff -- maybe 'Back Middle' so it's more parallel with the `current` callouts?

Reply:
Supposedly the position is fairly precise on the rivets, which is why I used it.

Reply:
In specific, the player in the middle must be nearest to the cone player.

Reply:
What about:

Front Left,
Front Right,
Front of Middle
Middle, Behind Current